apps/temp-docs/package.json

```diff
@@ -9,12 +9,14 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@apollo/client": "^3.6.6",
```

Comment:
any particular reason for apollo over octokit? https://github.com/octokit/graphql.js/

Reply:
There is no reason for apollo over octokit, but at the time it was easier to for my first time using graphql. Though we can always switch to octokit.